JSDemos/menuMeta.json

```diff
@@ -788,7 +788,6 @@
         "Demos": [
           {
             "Title": "Overview",
-            "Hidden":  true,
             "Name": "ExportToPDF",
             "DocUrl": "Guide/Widgets/DataGrid/Client-Side_Exporting/",
```

Comment:
Let's remove DocUrl property. The documentation not ready yet.

Reply:
done